public/css/room-directory.css

```
@@ -409,3 +429,24 @@
   background-color: #2774c2;
   color: #ffffff;
 }
+
+.RoomDirectoryView_safeSearchInput {
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  -o-appearance: none;
+  appearance: none;
+  position: relative;
+  width: 40px;
+  height: 20px;
+  background-color: #ffffff;
+  border: 1px solid rgba(200, 200, 200, 1);
+  box-shadow: inset -20px 0px 0px 0px rgba(200, 200, 200, 1);
+  border-radius: 10px;
+  transition: all 0.2s ease;
+  right: 0;
+}
```

I haven't checked this out locally to see if these styles are even relevant but I would reformat what's here to:

```suggestion
.RoomDirectoryView_safeSearchInput {
  position: relative;
  right: 0;
  width: 40px;
  height: 20px;
  
  appearance: none;
  background-color: #ffffff;
  border: 1px solid rgba(200, 200, 200, 1);
  box-shadow: inset -20px 0px 0px 0px rgba(200, 200, 200, 1);
  border-radius: 10px;
  transition: all 0.2s ease;
}
```